File: eleicoes/eleicoes.py
#!/bin/env python
#-*-encoding: utf-8 -*-
count = {}
winner = 0
winner_points = 0
size = int(input())
if size > 0 and size < 1000000000:
		for i in range(size):
				vote = int(input())
				if vote > 0 and vote < 1000000000:
						if not count.has_key(vote):
								count[vote] = 0
						else:
								count[vote] += 1
								if count[vote] > winner_points:
										winner = vote
										winner_points = count[vote]

print winner

File: eleicoes/eleicoes.rb
#!/bin/env ruby
count = {}
winner = 0
winner_points = 0
size = gets.to_i
if size > 0 and size <= 100000
  size.times do
    vote = gets.to_i
    if vote > 0 and vote <= 1000000000
      if count[vote.to_s.to_sym]
        count[vote.to_s.to_sym] += 1
      else
        count[vote.to_s.to_sym] = 1
      end
      if count[vote.to_s.to_sym] > winner_points
        winner = vote
        winner_points = count[vote.to_s.to_sym]
      end
    end
  end
end
puts winner
